janitor/functions/move.py

```diff
@@ -47,31 +46,37 @@ def move(
         0  2  w
         3  8  z
 
-        Moving a column
-
+        Move a column:
         >>> import pandas as pd
         >>> import janitor
-        >>> df = pd.DataFrame({"a": [2, 4, 6], "b": [1, 3, 5], "c": [7, 8, 9]})
+        >>> data = [{'a': 1, 'b': 1, 'c': 1,
+        ...          'd': 'a', 'e': 'a',
+        ...          'f': 'a'}]
+        >>> df = pd.DataFrame(data)
         >>> df
-           a  b  c
-        0  2  1  7
-        1  4  3  8
-        2  6  5  9
+           a  b  c  d  e  f
+        0  1  1  1  a  a  a
         >>> df.move(source="a", target="c", position="after", axis=1)
-           b  c  a
-        0  1  7  2
-        1  3  8  4
-        2  5  9  6
-        >>> df.move(source = 'c', target=None, position='before', axis=1)
-           c  a  b
-        0  7  2  1
-        1  8  4  3
-        2  9  6  5
-        >>> df.move(source = 'b', target=None, position='after', axis=1)
-           a  c  b
-        0  2  7  1
-        1  4  8  3
-        2  6  9  5
+           b  c  a  d  e  f
+        0  1  1  1  a  a  a
+        >>> df.move(source='f', target='b', position='before', axis=1)
+           a  f  b  c  d  e
+        0  1  a  1  1  a  a
+        >>> df.move(source='a', target=None, position='after', axis=1)
+           b  c  d  e  f  a
+        0  1  1  a  a  a  1
```

Comment:
nit: Just a little messy with a single example section having two kinds of quote-styles. I'll rather we stick to one, I'm partial to double-quotes since that is what black uses, but consistency is key.

![image](https://user-images.githubusercontent.com/30731072/219828380-1c3d1bc3-37ba-4dfa-b64d-eef3d8e6d571.png)

Also, I'ld also place the final `'f': 'a'` on the same line as the other two 'a' columns.